lib/libc/stdlib/malloc/Makefile.inc

```diff
@@ -10,6 +10,8 @@ MISRCS+=	malloc_revoke.c
 
 .if ${MK_MALLOC_REVOCATION_SHIM} == yes
 .include "${LIBC_SRCTOP}/stdlib/malloc/mrs/Makefile.inc"
+.else
+SYM_MAPS+=${LIBC_SRCTOP}/stdlib/malloc/Symbol_v3.map
```

Comment:
These shouldn't have default symbols at all (and that shouldn't appear in the map files). Fixing that upstream is on my pre-15 todo list, but we'll see when this happens.

Reply:
So would you rather me just remove these symbols from the map?

Reply:
I think that's safe to do in CheriBSD and keeps the diff smaller so we might as well.

Reply:
Done